deploy/examples/sharedserviceplan.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: "aerogear.org/v1alpha1"
+kind: "SharedServicePlan"
+metadata:
+  name: "keycloak-slice-plan"
+spec:
+  service: keycloak
+  name: keycloak slice
+  id: keycloak-default-slice
+  description: Slice of shared Keycloak service
+  free: true
+  bind_params:
+    $schema: http://json-schema.org/draft-04/schema#
+    type: object
+    properties:
+      Username: 
+        title: User name
```

Comment:
Can we have this as `User Name` just to keep consistent with the format of `Client Type`